### DISCUSSION TITLE
can't run yarn: Could not write file "/sandbox/yarn-error.log": "ENOSPC: no space left on device, write"

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [codesandbox/codesandbox-client#5646] [General] [answered]
https://codesandbox.io/s/github/NullVoxPopuli/ember-new-output/tree/stable?file=/package.json

![image](https://user-images.githubusercontent.com/199018/114194044-162bc280-991d-11eb-9d29-c63a3929ae9a.png)

**Comment by smhutch** [comment, marked as answer]
@NullVoxPopuli I had a look into this for you today. I couldn't identify the reason for the container being out of space, but found you could work around the issue by updating the node version used to 14.

To do this in your sandbox, please see [these docs](https://codesandbox.io/docs/faq#can-i-change-the-node-version-used-in-a-container-sandbox).

Please let me know if this resolves the issue for you. Thanks!

**Reply by NullVoxPopuli** [reply]
I stripped most of the dependencies from the template, and it would no longer run in to the issue. But, if I add all the lint dependencies again, I'll run in to the error. 1GB is not enough :-\

**Reply by NullVoxPopuli** [reply]
or wait, no.. I'm thinking of a different branch. I need to do some investigating. standby

**Reply by NullVoxPopuli** [reply]
got this just now:
![image](https://user-images.githubusercontent.com/199018/118864487-0a350800-b8ae-11eb-9be8-ce79247cfb29.png)

however, I had node 14 in the wrong place in the config, so I need to wait for the container to tear itself down to try again

**Reply by smhutch** [reply]
@NullVoxPopuli did you try restarting the sandbox (and/or the server)?

Here's where you can find that in the UI:
<img width="328" alt="CleanShot 2021-05-20 at 12 57 41@2x" src="https://user-images.githubusercontent.com/7217244/118974713-0adfa400-b96b-11eb-9c46-0f4c7f922bd9.png">

**Reply by NullVoxPopuli** [reply]
templates don't have those buttons
![image](https://user-images.githubusercontent.com/199018/118977417-1ec3df80-b944-11eb-88ca-49c71ac2c2f3.png)

🤔 I forked and it booted just fine: https://codesandbox.io/s/zen-hooks-oc3eu
Now I'm confused.
Why wouldn't the regular template just run?: https://codesandbox.io/s/github/NullVoxPopuli/ember-new-output/tree/stable?file=/package.json
(though, by the time anyone sees this, it will have cleaned itself up, and who knows what state it'll be in then 🙃 )

**Comment by lbogdan** [comment]
@NullVoxPopuli I deleted that template's persistent volume and restarted it, now it only uses ~350MB and it's working fine.

**Reply by NullVoxPopuli** [reply]
oh thanks! I'll mark this as resolved for now, and comment back if I run in to issue. It's good to know that the template is only 350mb
